src/Components/src/Microsoft.AspNetCore.Components/Microsoft.AspNetCore.Components.csproj

```diff
@@ -7,6 +7,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.JSInterop" Version="$(JSInteropPackageVersion)" />
+    <PackageReference Include="Microsoft.AspNetCore.Html.Abstractions" Version="$(MicrosoftAspNetCoreHtmlAbstractionsPackageVersion)" />
```

Comment:
As we've discussed, expecting this reference to be removed before this is merged.